package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-plaid-link-sdk",
-  "version": "7.2.0",
+  "version": "7.2.1",
```

Comment:
Let's update the package.json in the example too

Reply:
Will update once this is actually published